test/e2e/specs/CategoriesBetaPage.spec.js

```diff
@@ -0,0 +1,35 @@
+describe('Testing the Categories Beta page elements', () => {
+	it('Wait for all GraphQL requests to finish', () => {
+		// Spy on every GraphQL request
+		cy.intercept('POST', '**/graphql*').as('graphqlRequest');
+
+		// Go to all categories beta page
+		cy.visit('/categories-beta');
+
+		// Wait for page buttons to exist
+		cy.get('@graphqlRequest.all');
+		cy.findAllByText('View all loans');
+		cy.findAllByText('Get started');
+		cy.wait(1000);
```

Comment:
Please remove this wait. The test is probably failing without it because of the query after the page loads to fetch the loan spotlight, but waiting an arbitrary amount of time makes this test very fickle and it will fail if that query takes longer than 1 sec. 

A better approach is to wait on the graphql query for the loan spotlight, see the example here: https://github.com/kiva/ui/blob/main/test/README.md about waiting for aliased, multiple queries.